### PR TITLE
Add PingAn Cloud Support

### PR DESCRIPTION
**Pull request by kingsd041**

### CODE REVIEW

File: scripts/install2disk [hunk 1]
```diff
@@ -114,6 +114,51 @@ EOF
...
+    "pingan-ecs")
+        cat >cloud-config.yml<<EOF
+#cloud-config
+write_files:
+  - path: /etc/rc.local
+    permissions: "0755"
+    owner: root
+    content: |
+      #!/bin/bash
+      METADATA_SERVER="100.68.11.8"
```
Comment (Jason-ZW) [inline scripts/install2disk:126]: It is better to pass in environment variables

**Review by Jason-ZW — APPROVED** [review]
Others LGTM

File: scripts/install2disk (hunk 1 above)
Comment (kingsd041) [reply scripts/install2disk:126]: This is a fixed IP

**Review by Jason-ZW — APPROVED** [review]
LGTM

**Pull request merged**